scss/tech.scss

```diff
@@ -51,7 +52,15 @@
 
 
 .description {
-  @extend %size-letter-medium;
-  margin: 15px;
+  margin: 15px 9px 0 5px;
   text-align: center;
 }
+
+.layout {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: 10px 0 45px 125px;
+  padding: 10px 600px 0 0;
+  position: relative;
```

Comment:
I think 600 px of padding is a really big number. I'd recommend to add  padding to both sides and in a minor amount